### PR TITLE
Merge dev 1.x

### PR DESCRIPTION
**Pull request by evansiroky**
This PR brings in everything that was committed to OTP dev-1.x since the around the time the TriMet MOD project started in mid-2018.

### CODE REVIEW

File: docs/Deployments.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ The following are known deployments of OTP in a government- or agency-sponsored
 * **Atlanta, Georgia** The Metropolitan Atlanta Rapid Transit Authority's (MARTA) [trip planner](http://itsmarta.com/planatrip.aspx) and the Atlanta region's transit information hub [atltransit.org](https://atltransit.org/) both use OTP to power their website trip planners.
 * **Boston, Massachusetts** The [Massachusetts Bay Transportation Authority trip planner](https://www.mbta.com/trip-planner).
 * **Seattle, Washington** The [Sound Transit Trip Planner](https://www.soundtransit.org/tripplanner) is based on OTP. OTP also powers the trip planning feature of the [OneBusAway native apps](http://onebusaway.org/) in the Puget Sound region.  Technical details are [here](https://github.com/OneBusAway/onebusaway-android/blob/master/SYSTEM_ARCHITECTURE.md#add-trip-planning-andor-bike-share-optional).
-* **Arlington, Virginia** The [commute planning site](http://www.carfreeatoz.com/) for the Washington, DC metropolitan area depends on OpenTripPlanner to weigh the costs and benefits of various travel options, making use of profile routing.
```
Comment (landonreed) [inline docs/Deployments.md:15]: :(

File: docs/Developers-Guide.md [hunk 2]
```diff
@@ -242,61 +247,125 @@ is pushed to the main OpenTripPlanner repository on GitHub, this server will com
...
-- AWS IAM credentials
-- Sonatype-OSS staging / Maven artifact deployment
-- Merging dev branches into master ("git flow")
-- Release from master should happen _automatically_ once version is changed
-- ONLY one commit must have a particular non-snapshot commit (the tagged commit that constitutes the release)
-- Add final steps: `git checkout dev-1.x`; `git merge master` (primarily to update the POM version)
+The main OTP user documentation is built from Markdown files in the `/docs` directory of the repo. 
 
+The REST API docs and Javadoc are built by Maven, then uploaded manually to AWS S3, from which they are served as a web
+page at dev.opentripplanner.org. The main OTP user documentation is built by Readthedocs and served at docs.opentripplanner.org.
```
Comment (landonreed) [inline docs/Developers-Guide.md:368]: Do we need to update our branch to use our URL?

File: src/main/java/org/opentripplanner/graph_builder/module/osm/OSMFilter.java [hunk 3]
```diff
@@ -26,23 +26,30 @@
...
-                            .equals("unbuilt")))
-            return false;
+        if (highway != null) {
+            if(
+                    highway.equals("conveyer") ||
+                    highway.equals("proposed") ||
+                    highway.equals("construction") ||
+                    highway.equals("razed") ||
+                    highway.equals("raceway") ||
+                    highway.equals("unbuilt")
```
Comment (landonreed) [inline src/main/java/org/opentripplanner/graph_builder/module/osm/OSMFilter.java:42]: Wow, much improved formatting haha

File: src/main/java/org/opentripplanner/profile/RoundBasedProfileRouter.java [hunk 4]
```diff
@@ -17,11 +17,11 @@
 import com.google.common.base.Predicate;
 import com.google.common.collect.Collections2;
 
-import org.mapdb.Fun.Tuple2;
```
Comment (landonreed) [inline src/main/java/org/opentripplanner/profile/RoundBasedProfileRouter.java:20]: Mapdb!

**Review by landonreed — APPROVED** [review]
Generally, LGTM. I suppose we should set up a GTFS-flex instance with this to verify that things are still working there. Let's make merge dependent on getting the Vermont instance working (@binh-dam-ibigroup can you be in charge of that?).

**Comment by evansiroky** [comment]
Unassigning myself for now as it seems no further action is needed from me at the moment.

**Comment by landonreed** [comment]
Going to go ahead and merge (we can handle flex testing later).

**Pull request merged**